src/index.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-env browser */
+import units from "units-css";
+
+const DEFAULT_CANVAS = document.createElement("canvas");
+const DEFAULT_FONT_WEIGHT = 400;
+const DEFAULT_FONT_STYLE = "normal";
+
+const measureHeight = (size, lineHeight) => {
+  // If the line-height is unitless,
+  // multiply it by the font size.
+  if (!lineHeight.unit) {
+    return units.parse(
+      `${size.value * lineHeight.value}${size.unit}`
+    );
+  }
+
+  // units-css requires the user to provide
+  // DOM nodes for these units. We don't want
+  // to pollute our API with that for the time being.
```

Comment:
It seems an important use case to support `em` line-height inputs. If you agree, please create an issue.